lib/query-interface.js

```diff
@@ -537,7 +537,9 @@ class QueryInterface {
     options = _.clone(options) || {};
     options.type = QueryTypes.INSERT;
     const sql = this.QueryGenerator.bulkInsertQuery(tableName, records, options, attributes);
-    return this.sequelize.query(sql, options);
+    return this.sequelize.query(sql, options).then(results => {
+      return _.head(results);
```

Comment:
`return this.sequelize.query(sql, options).get(0)` http://bluebirdjs.com/docs/api/get.html